src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import { useState, FormEvent } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User } from './types/todo';
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todoList, setTodoList] = useState<Todo[]>(todos);
+
+  const [title, setTitle] = useState<string>('');
+  const [selectedUserId, setSelectedUserId] = useState<number>(0);
+
+  const [titleEror, setTitleEror] = useState<string>('');
```

Comment:
There is a typo in the variable name `titleEror`. It should be `titleError` to maintain consistency and clarity.